### PR TITLE
Add bazel-verify step to sigs.k8s.io/cluster-api

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

### CODE REVIEW

**Comment by vincepri** [comment]
Related to https://github.com/kubernetes-sigs/cluster-api/issues/43

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/cluster-api:v20180927-f09bc3b1b
+        args:
+        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=execute"
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml]: Note: execute scenario jobs should eventually move to the pod-utils instead, it's less config!

File: testgrid/config.yaml [hunk 2]
```diff
@@ -2445,6 +2445,9 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-org-verify-all
   num_columns_recent: 30
 #presubmits
+- name: pull-cluster-api-bazel-verify
```
Comment (BenTheElder) [inline testgrid/config.yaml]: this needs to be added to one of the dashboards (cluster lifecycle?)

Comment (vincepri) [reply testgrid/config.yaml]: Done : )

**Pull request merged**